ibm/mas_devops/playbooks/dependencies/install-bas.yml

```diff
@@ -5,8 +5,8 @@
     # BAS Configuration
     bas_namespace: "{{ lookup('env', 'BAS_NAMESPACE') | default('ibm-bas', true) }}"
 
-    bas_persistent_storage_class: "{{ lookup('env', 'BAS_PERSISTENT_STORAGE_CLASS') }}"
-    bas_meta_storage_class: "{{ lookup('env', 'BAS_META_STORAGE_CLASS') }}"
+    bas_persistent_storage_class: "{{ lookup('env', 'BAS_PERSISTENT_STORAGE_CLASS') | default('ibmc-file-bronze-gid') }}"
```

Comment:
You can't do this .. as I mentioned before .. this playbook is not specific to ROKS, these classes only exist on ROKS.  

- If a user calls this playbook directly they have to choose a storage class.  
- If a user calls this playbook via the various `-roks` playbooks then the class will have been set.

Unless the playbook has `-roks` in it's name there can be no assumptions about ROKS storages classes being available.

Reply:
fixed, removed default storage classes for ROKS only. made a small correction to the docs as well as it's not going to be used per default anymore.